src/Apps/Conversation/Components/ConversationSnippet.tsx

```diff
@@ -32,59 +39,78 @@ const ConversationSnippet: React.FC<ConversationSnippetProps> = props => {
   const item = conversation.items[0].item
 
   let imageURL
-  let title
+
   if (item.__typename === "Artwork") {
     imageURL = item.image && item.image.url
-    title = item.title
   } else if (item.__typename === "Show") {
     imageURL = item.coverImage && item.coverImage.url
-    title = item.name
   }
-  const date = DateTime.fromISO(conversation.lastMessageAt).toRelative()
 
   const partnerName = conversation.to.name
 
   const conversationText =
     conversation.lastMessage && conversation.lastMessage.replace(/\n/g, " ")
 
+  const truncatedText = truncate(conversationText, 100).html
+
   return (
-    <Box py={1}>
-      <StackableBorderBox
-        flexDirection="row"
-        alignItems="center"
-        justifyContent="space-between"
+    <Box>
+      <Link
+        href={`/user/conversations/${conversation.internalID}`}
+        underlineBehavior="none"
       >
-        <Flex>
-          <Flex height="auto" alignItems="center" mr={2}>
-            <Image src={imageURL} width="55px" />
-          </Flex>
-          <Flex flexDirection="column" justifyContent="center">
-            <Link
-              href={`/user/conversations/${conversation.internalID}`}
-              underlineBehavior="hover"
-            >
-              <Flex flexDirection="row">
-                <Serif size="2">{partnerName}</Serif>
-                <Serif size="2" ml={0.5} mr={0.5}>
-                  -
-                </Serif>
-                <Serif italic size="2" color="black60">
-                  {title && title.trim()}, {date}
-                </Serif>
-              </Flex>
-            </Link>
-            {conversationText}
+        <Flex alignItems="center" px={1} width="100%" height="120px">
```

Comment:
Is the `height` needed here?

Reply:
Its so I can center the items inside it vertically